templates/zerver/help/user-groups.md

```diff
@@ -32,7 +29,33 @@ any user group they are a part of. Guests cannot modify user groups.
 
 {end_tabs}
 
+### Configure who can create and manage user groups
+
+{!admin-only.md!}
+
+By default, [all members](/help/roles-and-permissions) in a Zulip
+organization can create user groups and manage user groups that they
+are a member of. However, you can restrict that ability to specific
+[roles](/help/roles-and-permissions).
+
+Note that administrators and moderators can modify any user group,
+while full and new members' ability to modify user groups is
+limited to ones they are a member of.
```

Comment:
How about something like this?

> Note that administrators and moderators can modify any user group,
while other organization members can only modify user groups to which they belong.